rpc/cloudagent/security/keyvault/key/moc_cloudagent_key.proto

```diff
@@ -34,6 +35,7 @@ message KeyOperationResponse {
 	string Data = 1 [(sensitive) = true];
 	google.protobuf.BoolValue Result = 2;
 	string  Error = 3;
+	Key key = 4;
```

Comment:
Is this the private key we are passing back?

Reply:
We should ensure the private key is not going out of MOC

Reply:
It's the same structure as the one returned by `Get` which does not include the private key. We really only need the `keyAge` and the `keyVersion`. Let me modify the structure to return just the fields necessary.